daemon/algod/api/algod.oas2.json

```diff
@@ -358,7 +358,7 @@
         }
       }
     },
-    "/v2/status/wait-for-block-after/{round}/": {
+    "/v2/status/wait-for-block-after/{round}": {
```

Comment:
When I attempted to make the call with extra "/", it would return me 404. It looks like something related to Echo, but I didn't spent too much time here, as I've figure that we want it to align with the rest of the REST commands. ( i.e. which doesn't have trailing "/" )

Reply:
There is a "strip removing slash" feature in echo, implemented as a "Pre" middleware (as in before routing is done). I think whats happening here is the router requires the trailing slash, but the pre-router middleware removes it. So basically there's no way to actually reach the endpoint when there is a slash.

Nice catch.